clinvar/src/main/scala/org/broadinstitute/monster/etl/clinvar/ClinvarMapper.scala

```diff
@@ -4,6 +4,7 @@ import cats.data.NonEmptyList
 import com.spotify.scio.coders.Coder
 import com.spotify.scio.values.SCollection
 import org.broadinstitute.monster.etl.MsgTransformations
+import org.broadinstitute.monster.etl.clinvar.ClinvarConstants.IdKey
```

Comment:
Can you replace this import with a relative import nested within the `ClinvarMapper` class? i.e.
```scala
class ClinvarMapper {
  import ClinvarConstants.IdKey

  ... existing code ...
}
```
It's a code style to distinguish class / object imports from field imports.

Reply:
Can you also replace the existing uses of `ClinvarConstants.IdKey` with just `IdKey` for consistency?